src/Storages/MergeTree/IMergeTreeDataPart.cpp

```diff
@@ -888,25 +890,23 @@ void IMergeTreeDataPart::loadColumns(bool require)
         /// If there is no file with a list of columns, write it down.
         for (const NameAndTypePair & column : metadata_snapshot->getColumns().getAllPhysical())
             if (volume->getDisk()->exists(getFullRelativePath() + getFileNameForColumn(column) + ".bin"))
-                columns.push_back(column);
+                loaded_columns.push_back(column);
 
         if (columns.empty())
```

Comment:
Looks this should be `loaded_columns` - https://github.com/ClickHouse/ClickHouse/pull/64757